Command/DebugGatewayCommand.php

```diff
@@ -123,6 +124,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return Command::SUCCESS;
     }
 
+    /**
+     * @return array<int, string>
```

Comment:
This can be a list instead

```suggestion
     * @return list<string>
```